app/Http/Controllers/TransportController.php

```diff
@@ -96,6 +96,15 @@ public static function FastTripAccess($departure, $lineName, $number, $when)
         return null;
     }
 
+    public static function StationByCoordinates($latitude, $longitude)
+    {
+        $client = new Client(['base_uri' => config('trwl.db_rest')]);
+        $response = $client->request('GET', "stops/nearby?latitude=$latitude&longitude=$longitude&results=1");
```

Comment:
Maybe it would be beautiful to use "http_build_query()" function, but it's not neccessary.